modules/circe/src/test/scala/pureconfig/circe/CirceSuite.scala

```diff
@@ -0,0 +1,36 @@
+package pureconfig.module.circe
```

Comment:
Can you please move this to `modules/circe/src/test/scala/pureconfig/module/circe/CirceSuite.scala`?